extensions/sockets/pom.xml

```diff
@@ -39,5 +56,23 @@
             <artifactId>mockito-core</artifactId>
             <scope>test</scope>
         </dependency>
+        <dependency>
+            <groupId>org.assertj</groupId>
+            <artifactId>assertj-core</artifactId>
+            <version>3.3.0</version>
+            <scope>test</scope>
+        </dependency>
+        <dependency>
+            <groupId>org.mule.tests</groupId>
+            <artifactId>mule-tests-unit</artifactId>
```

Comment:
except for assertJ, all the other dependencies you actually inherit from the parent pom. You're not only being redundnat, but in some cases missing the test scope.